tests/test_encoder.py

```diff
@@ -488,3 +496,57 @@ def test_specific_alternates(encoder_context):
     alternates = encoder.decode_selected_alternates(alternates_by_index)
 
     assert alternates == alternates_by_forum
+
+
+def test_encoder_no_reviewers(encoder_context):
+    papers, reviewers, matrix_shape = encoder_context(n_reviewers=0)
+
+    scores_by_type = {
+        'mock/-/score_edge': {
+            'edges': [(forum, reviewer, 0.5) for forum, reviewer in itertools.product(papers, reviewers)]
+        }
+    }
+
+    weight_by_type = {
+        'mock/-/score_edge': 1
+    }
+
+    constraints = []
+
+    with pytest.raises(EncoderError) as exc:
+        encoder = Encoder(
+            reviewers,
+            papers,
+            constraints,
+            scores_by_type,
+            weight_by_type
+        )
+
+    assert 'Reviewers List is empty.' == str(exc.value)
```

Comment:
can you add a test where the solver returns an `Error` status and the text of the error in the description?

You would need to return an empty list in the openreview_interface mockup.